river/tree/_objective/binary_cross_entropy_objective.py

```diff
@@ -0,0 +1,16 @@
+import math
```

Comment:
Given the simplicity of the implementation of `BinaryCrossEntropyObjective` and `SquaredErrorObjective`, I suggest keeping them in the same file or even included alongside `BaseObjective`. This could improve the readability of these components.

Reply:
I moved all the classes to a single file in commit [940122d](https://github.com/online-ml/river/pull/458/commits/940122dfb81b6e62d7f43a2e51cfb113b90d899d). 

Ideally, we could think of other objective functions to add. In this sense, the objective function to use would a parameter of the tree.